### PR TITLE
Import problem fix: DCOS-50640

### PR DESCRIPTION
**Pull request by ahairape**
Applying the fix to https://jira.mesosphere.com/browse/DCOS-50640 proposed by @jieyu

### CODE REVIEW

**Comment by ahairape** [comment]
@jieyu @gracedo PTAL, thank you

**Comment by jieyu** [comment]
@ahairape can you make sure to not include the .DS_Store file?

File: dashboards/Secrets/Vault/Vault-Summary.json [hunk 1]
```diff
@@ -790,7 +822,7 @@
     ]
   },
   "timezone": "utc",
-  "title": "Secrets: Summary Vault",
+  "title": "/dcos-generic/dashboards/Secrets/Vault/Vault-Summary",
```
Comment (gracedo) [inline dashboards/Secrets/Vault/Vault-Summary.json]: Can we change this back to the standardized title format?

**Review by gracedo — CHANGES_REQUESTED** [review]
Just a small comment to change the dashboard title back to the standardized version. I'm +1 on this if you've tested this on a cluster and grafana loads successfully with the new dashboard!

**Review by gracedo — APPROVED** [review]
Not sure how to test this in dcos-monitoring since it's a fork; we'll have to test after merging.

File: dashboards/Secrets/Vault/Vault-Summary.json (hunk 1 above)
Comment (ahairape) [reply dashboards/Secrets/Vault/Vault-Summary.json]: The only test I did was to change the IDs and re-import manually the dashboards on the `soak113s` cluster from where they had been exported.

**Comment by gracedo** [comment]
I updated dcos-monitoring's grafana-dashboards submodule with the latest version (including the changes in this PR) and installed the stub (generated locally) on a cluster and didn't see any errors. I created a secret as a test and saw datapoints show up in the graphs.
image
image

**Pull request merged**